tools-v2/pkg/cli/command/curvebs/list/scanstatus/scanstatus.go

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
This command seems to have no output?

Reply:
> debian stretch-updates Release
> #0 3.234   404  Not Found

debian9 EOL，
https://stackoverflow.com/questions/76094428/debian-stretch-repositories-404-not-found